06_deploy_masters.sh

```diff
@@ -8,7 +8,12 @@ instack="ocp/master_nodes.json"
 export OS_TOKEN=fake-token
 export OS_URL=http://ostest-api.test.metalkube.org:6385/
 
-# Clean previously env 
+wait_for_json ironic \
```

Comment:
This doesn't work because we don't source utils.sh in this script atm.

Reply:
oops. Forgot that little step. I was trying it on the terminal by sourcing it
:-)